src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -43,7 +45,7 @@
 
 .row {
   display: grid;
-  grid-template-columns: $left_column $middle_column $middle_column_right_gap $right_column;
+  grid-template-columns: $left_column $middle_column_left_gap $middle_column $middle_column_right_gap $right_column;
```

Comment:
I may be missing something, but why did you decide to add another column to the grid:

![image](https://user-images.githubusercontent.com/6834224/92146675-f629a300-ee11-11ea-93e9-471fc2d412bc.png)


rather than increase the right padding of the "left" section (it already has some amount of right padding anyway):

<img src="https://user-images.githubusercontent.com/6834224/92146770-18232580-ee12-11ea-994a-b68c0917d877.png"  width="200"/>

Reply:
It is good to use the constants and keep the grid layout same as GeneOverview image (as per XD). If you look at the design you can see this column being aligned to the GeneOverview Image.

I left the padding there deliberately as it is being removed in a different PR (Replacing Default/Left text PR)